src/components/StyledInput.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { FC, HTMLAttributes, useState } from "react";
+
+import { StyleSheet, css } from "aphrodite";
+import { colors } from "styles/palette";
+
+interface Props {
+  value: string;
+  error?: string;
+  focused?: boolean;
+  onBlur?: () => void;
+  onFocus?: () => void;
+  label?: string;
+  placeholder?: string;
```

Comment:
I think you always set placeholder then is it good idea that it optional ?

Reply:
It can stay as it is